apps/nrm/behaviours/casework-submission.js

```diff
@@ -33,7 +33,7 @@ module.exports = conf => {
 
         const externalID = req.sessionModel.get('externalID')  || caseWorkPayload.ExternalId;
 
-        const reportReference = req.sessionModel.get('reference');
+        const reportID = req.sessionModel.get('id');
```

Comment:
It would be good to explain the purpose of report id and external id. Not sure where, maybe herr?

Reply:
Comment added